samples/uikit/build.gradle.kts

```diff
@@ -91,72 +91,18 @@ val packForXCode = if (sdkName == null || targetBuildDir == null || executablePa
     // Otherwise copy the executable into the Xcode output directory.
     tasks.create("packForXCode", Copy::class.java) {
         dependsOn(kotlinBinary.linkTask)
+
         destinationDir = file(targetBuildDir)
+
+        val dsymSource = kotlinBinary.outputFile.absolutePath + ".dSYM"
```

Comment:
Not sure if handling `.dSYM` is entirely correct here.
1. Checking if file exists on task configuration is too early.
2. Correct name is `sample.app.dSYM`, not `sample.dSYM`.
3. Isn't `rename` below applied to this copied `.dSYM` as well? How does this work?

Reply:
Fixed.